Content.Client/Entry/IgnoredComponents.cs

```diff
@@ -326,7 +326,8 @@ public static class IgnoredComponents
             "EnergySword",
             "DoorRemote",
             "InteractionPopup",
-            "HealthAnalyzer"
+            "HealthAnalyzer",
```

Comment:
This is fine but in the future it's easier to pick a random place in the middle to have less conflicts